frontend/interfaces/vulnerabilities.ts

```diff
@@ -0,0 +1,17 @@
+import PropTypes from "prop-types";
+
+export default PropTypes.shape({
+  id: PropTypes.number,
+  uid: PropTypes.number,
+  username: PropTypes.string,
+  type: PropTypes.string,
+  groupname: PropTypes.string,
+});
+
+export interface IVulnerabilities {
```

Comment:
Perhaps clearer to do this as the usage of `IVulnerabilities[]` makes me think this is intended to be the interface for a single vulnerability?
```suggestion
export interface IVulnerability {
```